.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: "CodeQL"
+on:
+  push:
+    branches: [ "develop", fleek, fleek-pre, master ]
```

Comment:
use quotes for all of them

Reply:
Yes, this is an automatic update of the "database" of dependencies.